index.js

```diff
@@ -67,7 +68,144 @@ function getDefaultConfig(mainTemplate, templatesPath) {
 
       // Add extra function for finding a security scheme by name
       ramlObj.securitySchemeWithName = function (name) {
-        return ramlObj.securitySchemes.find((scheme) => scheme[name]);
+        for (var index = 0; index < ramlObj.securitySchemes.length; ++index) {
```

Comment:
A version that works, return the value of the scheme object, rather then the object itself. I already fixed it in develop by the way.

```
    ramlObj.securitySchemeWithName = function (name) {
        const result = ramlObj.securitySchemes.find(s => s[name]);
        if (result) {
          return result[name];
        }
        return {};
      };
```